charts/argocd-config/templates/app-of-apps.yaml

```diff
@@ -9,9 +9,9 @@ spec:
   project: {{ .Values.environment }}
   source:
     path: charts/argocd-apps
-    repoURL: https://wbstack.github.io/charts
-    targetRevision: 1.0.1
-    chart: argocd-apps
+    repoURL: https://github.com/wbstack/charts.git
+    targetRevision: api-argocd # debug: should be HEAD
+    #chart: argocd-apps
```

Comment:
This file should not be changed in this PR; we should do this in a seperate PR.

It should also just bump the chart version and it should continue using the helm repo style repo rather than a git one.

After this change we will also need to increase the argocd-config chart version

Reply:
done https://github.com/wbstack/charts/pull/160